### ISSUE TITLE
Saved movies no longer being auto-deleted as per settings

### ISSUE THREAD

**Issue by NginUS** [ccrisan/motioneye#1275]
I'm on Linux Mint 18.3 (Ubuntu 16.04 LTS)
motionEye Version | 0.40
Motion Version | 4.2.2

Several days ago I lost all my configuration & had to start over, detailed in issue 1261 "Timestamp Issues".

What's happening now since then is that although I have the 'Movies' section of the MotionEye settings set to a 'Custom' retention threshold of 14 days [1], the WebUI's Movies Browser displays [2] 15-day old videos. The only sense I can make of this is that maybe it doesn't count the current day when it decides what to delete?

Furthermore, when I browse that directory in my File Manager, it shows [3] there being a few even older files that were recorded far beyond the 15-day limit- oddly for just 2 days with a significant gap. Note that these aren't made available through the WebUI's Movie Browser. They're owned by root same as the ones that are displayed, so I don't know what sense to make of that.

[1]
https://files.catbox.moe/ifiktp.png
![Settings](https://user-images.githubusercontent.com/8769067/57616630-9c320980-754c-11e9-9316-041bf681ff2b.png)

[2]
https://files.catbox.moe/npbxpp.png
![WebUI_Movies_Browser](https://user-images.githubusercontent.com/8769067/57616649-a7853500-754c-11e9-91b3-652c2dfb8f36.png)

[3]
https://files.catbox.moe/worlg9.png
![Save_Path_Contents](https://user-images.githubusercontent.com/8769067/57616665-b23fca00-754c-11e9-83cd-714a033b2765.png)

**Comment by ccrisan** [comment]
Could you please attach your motioneye log here so we can see what the cleanup procedure has to say?

**Comment by NginUS** [comment]
[motioneye.log](https://github.com/ccrisan/motioneye/files/3182389/motioneye.log)

FYI: I've since manually deleted the problem files/directories. Here's a log anyway- doesn't look so helpful though. :-(

**Comment by NginUS** [comment]
I've changed the retention to 1 week, & waited to see what it would do.

What's happening now is that its deleting week-old directories like it should, except only the ones its created since I reconfigured everything. I wound up with a gap- some old ones remained that were well over a week old, but earlier week-old ones were deleted. Now I've manually deleted those very old ones which it wouldn't delete & all is well.
